torchtext/experimental/datasets/sequence_tagging.py

```diff
@@ -27,16 +27,16 @@ def _setup_datasets(dataset_name,
                     root=".data",
                     vocabs=None,
                     data_select=("train", "valid", "test")):
-    if isinstance(data_select, str):
-        data_select = [data_select]
-    if not set(data_select).issubset(set(("train", "valid", "test"))):
-        raise TypeError("Given data selection {} is not supported!".format(data_select))
-
-    train, val, test = raw.DATASETS[dataset_name](root=root)
+    data_select = check_default_set(data_select, ('train', 'valid', 'test'))
```

Comment:
This should be special-cased for CoNLL2000Chunking and eventually CoNLL2000Chunking will just pass that in directly